notes/storage-user-view.md

```diff
@@ -0,0 +1,21 @@
+**Note** This is a first attempt at recommendations for users.  It's unclear if these belong in this document or not.
+
+### Making Storages Discoverable
+
+A WebID owner MAY have zero or more storage spaces associated with the WebID and MAY choose, for each one, to make it discoverable by the general public, by a selected audience, or not at all. If a storage space is discoverable by someone, that only means they can know it exists.  It does not imply that they have read or write access to it - those permissions are defined elsewhere.
+
+WebID owners who wish their storage spaces to be discoverable by the general public, SHOULD advertise the space publicly by having a triple like this one in their profile document or in a publicly accessible extended profile document.
+```
+   <#me> <http://www.w3.org/ns/pim/space#storage> </> .
+```
+The object of the triple may be on the same host as the document the triple occurs in or on a different host.
+
+A storage triple is usually created by the pod provider on pod creation but if it is missing, the WebID owner SHOULD create it if they want the storage discoverable.  
+
+A WebID owner wishing to disallow the public from discovering a storage SHOULD NOT use pim:storage predicates for that storage in any publicly accessible document and should remove any such statements put there by a pod provider.  
+
+A WebID owner wishing to make a private storage space available to selected Agents, SHOULD create a pim:storage triple in a document only accessible by those Agents.
```

Comment:
This is going to be available only once Solid has triple level access rights?

Reply:
Thank you for clarifying.

Reply:
No, I didn't mean what either of you say.  If you put <#me> pim:storage </Foo/> in the preferences file, the only agents with access to the preferences file (private by default) will be able to see the URL and know that the WebID is associated with </Foo/>.  It *does* hide the URL.  This does not depend on triple-level privacy but on the private nature of the preferencesFile.  One could also use owl:sameAs to point to a document that is only accessible to four people and in that document put a pointer to a pim:Storage that only you and those four people will be able to discover exists.

Reply:
So the URL of the preferencesFile or sameAs is public but, if those documents are private the URLs of any storages mentioned in them will also be private.

Reply:
In that case, can we reword?

```suggestion
A WebID owner wishing to make a private storage space discoverable to selected Agents, SHOULD create a pim:storage triple in a separate document (such as the `preferencesFile`) only accessible by those Agents.
```

But I wouldn't agree to that mechanism, for the reason that:

> So the URL of the preferencesFile or sameAs is public but, if those documents are private the URLs of any storages mentioned in them will also be private.

No, only discovery would be private.

Reply:
Ah, yes, that's true.  The discovery of the URL would be private.  The document that the URL points to might be public or private.

I think I will try for an expanded paragraph to clarify, keeping in mind the points the two of you have made.  Will upload later.

Reply:
I went with this wording, @theRealImy  and @RubenVerborgh please let me know if it works for you.
```
A WebID owner wishing to make a storage space undiscoverable by the public but discoverable by selected Agents, SHOULD create a pim:storage triple in a document only accessible by those Agents. For example if the storage triple is in the pim:preferences document, it will be undiscoverable by the public. If the storage triple is in an owl:sameAs document restricted to three people, only those three people will be able to discover the storage.
```

Reply:
That is clearer. Thanks.